include/z64effect.h

```diff
@@ -258,4 +262,180 @@ typedef enum EffectSsType {
 #undef DEFINE_EFFECT_SS
 #undef DEFINE_EFFECT_SS_UNSET
 
+void EffectBlure_AddVertex(EffectBlure* this, Vec3f* p1, Vec3f* p2);
+void EffectBlure_AddSpace(EffectBlure* this);
+void EffectBlure_Init1(void* thisx, void* initParamsx);
+void EffectBlure_Init2(void* thisx, void* initParamsx);
+void EffectBlure_Destroy(void* thisx);
+s32 EffectBlure_Update(void* thisx);
+void EffectBlure_Draw(void* thisx, struct GraphicsContext* gfxCtx);
+void EffectShieldParticle_Init(void* thisx, void* initParamsx);
+void EffectShieldParticle_Destroy(void* thisx);
+s32 EffectShieldParticle_Update(void* thisx);
+void EffectShieldParticle_Draw(void* thisx, struct GraphicsContext* gfxCtx);
+void EffectSpark_Init(void* thisx, void* initParamsx);
+void EffectSpark_Destroy(void* thisx);
+s32 EffectSpark_Update(void* thisx);
+void EffectSpark_Draw(void* thisx, struct GraphicsContext* gfxCtx);
+void func_80026230(struct PlayState* play, Color_RGBA8* color, s16 arg2, s16 arg3);
+void func_80026400(struct PlayState* play, Color_RGBA8* color, s16 arg2, s16 arg3);
+void func_80026608(struct PlayState* play);
+void func_80026690(struct PlayState* play, Color_RGBA8* color, s16 arg2, s16 arg3);
+void func_80026860(struct PlayState* play, Color_RGBA8* color, s16 arg2, s16 arg3);
+void func_80026A6C(struct PlayState* play);
+struct PlayState* Effect_GetPlayState(void);
+void* Effect_GetByIndex(s32 index);
+void Effect_InitContext(struct PlayState* play);
+void Effect_Add(struct PlayState* play, s32* pIndex, s32 type, u8 arg3, u8 arg4, void* initParams);
+void Effect_DrawAll(struct GraphicsContext* gfxCtx);
+void Effect_UpdateAll(struct PlayState* play);
+void Effect_Delete(struct PlayState* play, s32 index);
+void Effect_DeleteAll(struct PlayState* play);
+void EffectSs_InitInfo(struct PlayState* play, s32 tableSize);
+void EffectSs_ClearAll(struct PlayState* play);
+void EffectSs_Delete(EffectSs* effectSs);
+void EffectSs_Reset(EffectSs* effectSs);
+void EffectSs_Insert(struct PlayState* play, EffectSs* effectSs);
+void EffectSs_Spawn(struct PlayState* play, s32 type, s32 priority, void* initParams);
```

Comment:
Effect and EffectSs are two totally different systems, even though they achieve basically the same thing. Not totally sure if it makes sense to combine them but my gut feeling says no

Reply:
but Id be fine with this being an intermediate step for the time being

Reply:
oh I didnt realize thats how it already was, alright. Yeah whenever we get to it